<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QMainWindow" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="user_name_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>63</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>用户名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="user_name_edit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>211</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="nick_name_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>63</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>昵称</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nick_name_edit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>211</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="pwd_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>63</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pwd_edit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>211</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pwd_confirm_edit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>211</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="pwd_confirm_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>63</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>确认密码</string>
    </property>
   </widget>
   <widget class="QLabel" name="mail_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>63</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>邮箱</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mail_edit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>260</y>
      <width>211</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="reg_btn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>330</y>
      <width>106</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QLabel" name="user_name_hint">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>用户名信息提示</string>
    </property>
   </widget>
   <widget class="QLabel" name="pwd_confirm_hint">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>210</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>密码信息提示</string>
    </property>
   </widget>
   <widget class="QLabel" name="mail_hint">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>270</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>邮箱信息提示</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>527</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
